cmd/metal-api/internal/service/tenant-service.go

```diff
@@ -8,6 +8,7 @@ import (
 	v1 "github.com/metal-stack/masterdata-api/api/rest/v1"
 	mdmv1 "github.com/metal-stack/masterdata-api/api/v1"
 	mdm "github.com/metal-stack/masterdata-api/pkg/client"
+	internalV1 "github.com/metal-stack/metal-api/cmd/metal-api/internal/service/v1"
```

Comment:
Maybe just call this API v1 as in the other packages and rename the existing v1 import to mdmv1rest or something.